docs/troubleshoot/verify-fingerprint.md

```diff
@@ -86,14 +89,19 @@ Note that you will not have a `fingerprint` directory in releases prior to v1.14
    -->
 
    **For Zowe releases prior to v1.14**
+
+   Issue commands similar to the following. In this example, you use Zowe v1.9.
+
    ```
    /u/username/hash/zowe-verify-authenticity.sh -r /usr/lpp/zowe/v1.9 -f /u/username/hash -h /u/username/hash
    ```
 
-The `zowe-verify-authenticity.sh` script creates a `CustRuntimeHash.txt` file, which it compares with the `RefRuntimeHash-v.r.m.txt` file.  
+The `zowe-verify-authenticity.sh` script creates a `CustRuntimeHash.txt` file, which it compares with the `RefRuntimeHash-V.v.p.txt` file.  
 
 ## Step 3: Review results
 
+There are two results:
```

Comment:
There are two possible outcomes, match or mismatch.  [note to Editor:  you can use any form of words here; I just want to explain that results will be either one or the other, not both, which "two results" implies]

Reply:
Changed to **You will get one of the following results.**